.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    - bugfix/**
+    - feature/**
```

Comment:
This project does not have `bugfix` or `feature` prefixes. Thus, we should not enforce it. Can be ditched IMHO.